themes/classic/templates/catalog/_partials/product-discounts.tpl

```diff
@@ -1,5 +1,5 @@
+{if $product.quantity_discounts}
```

Comment:
It will cause troubles for Ajax refreshing. We **NEED** to have the `div` to be able to put the infos in the right place if there are discounts for a specific combination.